src/CascadeMinimizer.cc

```diff
@@ -333,7 +336,7 @@ bool CascadeMinimizer::minimize(int verbose, bool cascade)
     }
 
     bool doMultipleMini = (CascadeMinimizerGlobalConfigs::O().pdfCategories.getSize()>0);
-    if ( doMultipleMini ) preFit_ = 1;
+    // if ( doMultipleMini ) preFit_ = 1;
```

Comment:
@nucleosynthesis i removed this as it didn't seem necessary to force the preFit, can you confirm it's ok to leave it out?